tuplex/core/include/DataSet.h

```diff
@@ -86,6 +86,16 @@ namespace tuplex {
 
         virtual ~DataSet();
 
+        /*!
+         * add a sort operation T -> S to the logical graph
+         * @param order std::vector<std::size_t> the order of columns (represented
+         * as indices) to sort by
+         * @param orderEnums std::vector<std::size_t> an enum corresponding to
```

Comment:
can we change size_t to an enum class which specifies the options explicitly?

Reply:
also, consider passing via `const std::vector&` instead of copy.

Reply:
Done.